src/main/res/values/strings.xml

```diff
@@ -153,11 +153,11 @@
   <string name="username_is_not_specified">Username missing</string>
   <string name="validation_index_error_description">Could not validate the index.</string>
   <string name="validation_package_error_description_format">Could not validate the package: %s.</string>
-  <string name="validation_error_hash_lower">Invalid checksum</string>
-  <string name="validation_error_integrity_lower">Could not check integrity</string>
-  <string name="validation_error_permissions_lower">Add required permissions</string>
-  <string name="validation_error_signature_lower">Invalid signature</string>
-  <string name="validation_error_structure_lower">Invalid file format</string>
+  <string name="validation_error_hash_lower">invalid checksum</string>
+  <string name="validation_error_integrity_lower">could not check integrity</string>
```

Comment:
"Could not validate the package: could not check integrity". I think "could not" used 2 times is weird.

Reply:
Went with generic style and changed to "Application" to avoid "package"

Reply:
It was about "could not" used 2 times, not about "application" :/ Using "application" is not correct in this place as well, should be "package". Don't mix these words, they aren't meaning the same thing.

Reply:
Hoping it is an "APK package".

Reply:
I mean, don't change "application" to "package". I used different words intentionally when it's needed. "APK package" is not good as well, because APK is "Android Package", and "APK package" is "android package package". I use "package" word when I'm talking about "package" (or about "android package").

Reply:
Gives nothing to search for is the issue I tried to avoid